contracts/LinniaRecords.sol

```diff
@@ -14,15 +14,17 @@ contract LinniaRecords is Ownable {
         uint sigCount;
         mapping (address => bool) sigs;
         uint irisScore;
-        // For now the record types are
-        // 0 nil, 1 Blood Pressure, 2 A1C, 3 HDL, 4 Triglycerides, 5 Weight
         uint recordType;
```

Comment:
consider doing one of the following:
1. removing recordType
2. Clearly defining why recordType is necessary